src/server/services/procedures/mortality/mortality.js

```diff
@@ -0,0 +1,228 @@
+/**
+ * This service accesses data from the human mortality database,
+ * which tabulates death rates broken down by age group and gender
+ * for various countries.
+ * 
+ * @alpha
+ * @service
+ * @category Health
```

Comment:
Maybe Science instead? It could be placed under "Demographic Studies" (probably nested under science) but as it would be the only service in that category, it probably should wait :)

Reply:
Yeah, Science is probably the closest fit considering the other services in that category. Akos just mentioned wanting at some point to have more health-oriented data sources, but they'd still technically be Science as well. In the new commit I've changed it to Science.

Reply:
Gotcha. Makes sense. When I first read "Health", I was imagining connecting to devices like a fitbit or something :)